src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -63,9 +63,16 @@ public class HubEndPoint<THub, TClient> : EndPoint where THub : Hub<TClient>
 
                 using (var scope = _serviceScopeFactory.CreateScope())
```

Comment:
Hubs are transient and a scope needs to be created whenever we make a hub. We'll need to see how bad the performance is for this.